ideascube/serveradmin/management/commands/catalog.py

```diff
@@ -149,6 +150,14 @@ def list_packages(self, options):
                 for pkg in pkgs:
                     print(fmt.format(pkg))
 
+        pkgs = self.catalog.list_nothandled(options['ids'])
+
+        if pkgs:
+            print('Not handled packages')
```

Comment:
Done on last "push --force"

By default, only display that x packages have been ignored.
With option --unhandled, the full list is displayed.